staging/src/k8s.io/apiserver/pkg/util/peerproxy/peerproxy_handler.go

```diff
@@ -129,121 +122,178 @@ func (h *peerProxyHandler) WrapHandler(handler http.Handler) http.Handler {
 			return
 		}
 
-		// StorageVersion Informers and/or StorageVersionManager is not synced yet, pass request to next handler
+		// Apiserver Identity Informers is not synced yet, pass request to next handler
 		// This will happen for self requests from the kube-apiserver because we have a poststarthook
-		// to ensure that external requests are not served until the StorageVersion Informer and
-		// StorageVersionManager has synced
+		// to ensure that external requests are not served until the ApiserverIdentity Informer has synced
 		if !h.HasFinishedSync() {
 			handler.ServeHTTP(w, r)
 			return
 		}
 
 		gvr := schema.GroupVersionResource{Group: requestInfo.APIGroup, Version: requestInfo.APIVersion, Resource: requestInfo.Resource}
-		if requestInfo.APIGroup == "" {
-			gvr.Group = "core"
-		}
-
-		apiservers, err := h.findServiceableByServers(gvr)
+		apiserversi, err := h.apiserverIdentityLeases()
 		if err != nil {
-			// resource wasn't found in SV informer cache which means that resource is an aggregated API
-			// or a CR. This situation is ok to be handled by local handler.
+			klog.ErrorS(err, "no apiserver identity leases found")
```

Comment:
Need an explanation why it's okay to serve locally if no apiserver identity leases are found. Logging level seems to indicate an error.

Reply:
Fixed